python/GangaLHCb/Lib/RTHandlers/RTHUtils.py

```diff
@@ -57,12 +57,6 @@ def lhcbdiracAPI_script_template():
 def is_gaudi_child(app):
     if isType(app, Gaudi):
         return True
-
-    if isType(app, TaskApplication):
-        from GangaLHCb.Lib.Applications import GaudiPythonTask, BenderTask
-        if not isType(app, GaudiPythonTask) and not isType(app, BenderTask):
```

Comment:
Sure, we can put this in for 6.1.19. The PR is against `develop` so we'll have to merge it there and then merge that into the release branch.